PoGo.Necrobot.Window/Model/PlayerInfoModel.cs

```diff
@@ -189,6 +196,48 @@ public void OnInventoryRefreshed(IEnumerable<InventoryItem> inventory)
                 .Where(x => x != null && x.Id == playerProfile.PlayerData.BuddyPokemon.Id)
                 .FirstOrDefault();
 
+            // get applied items
+            var appliedItems =
+                Session.Inventory.GetAppliedItems().Result
+                .SelectMany(aItems => aItems.Item)
+                .ToDictionary(item => item.ItemId, item => new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc).AddMilliseconds(item.ExpireMs));
+
+            var items =
+                     Session.Inventory.GetItems().Result
+                    .Where(i => i != null)
+                    .OrderBy(i => i.ItemId);
+
+            foreach (var item in items)
+            {
+                if (appliedItems.ContainsKey(item.ItemId))
+                {   
+                    switch (item.ItemId)
+                    {
+                        case ItemId.ItemLuckyEgg:
+                            Luckyeggs_expires = appliedItems[item.ItemId];
+                            break;
+                        case ItemId.ItemIncenseOrdinary:
+                            Insence_expires = appliedItems[item.ItemId];
+                            break;
+                        default:
+                            Insence_expires = appliedItems[item.ItemId];
+                            break;
+                    }
+                }
+            }
+
+            // add and enable timer
+            // 
+            // tmr
+            // 
+            tmr = new System.Timers.Timer()
```

Comment:
If Using System; is defined at the start, we don't need to reference it again

Meaning Just use Timers.Timer() instead